internal/campaigns/util.go

```diff
@@ -39,6 +51,18 @@ func IsKindSupported(extSvcKind string) bool {
 	return ok
 }
 
+func HasCodehostCapability(extSvcType string, capability CodehostCapability) bool {
```

Comment:
```suggestion
func ExternalServiceSupports(extSvcType string, capability CodehostCapability) bool {
```

(We use the `ExternalService` wording in the rest of the functionality here too)